lib/fog/core/service.rb

```diff
@@ -295,22 +273,6 @@ def camel_case_collection_name(collection)
         collection.to_s.split("_").map(&:capitalize).join
       end
 
-      # This will attempt to require all request files declared in the service using fog"s DSL
-      def require_requests_and_mock
-        requests.each do |request|
-          require_item(request, @request_path)
-          if service::Mock.method_defined?(request.last)
-            mocked_requests << request.last
-          else
-            service::Mock.module_eval <<-EOS, __FILE__, __LINE__
-              def #{request.last}(*args)
-                Fog::Mock.not_implemented
-              end
-            EOS
-          end
-        end
-      end
-
       # Requires the correct file for an item (collection, model, or request).
       #
       # @param [Array] item
```

Comment:
All the changes in this file. =)
We need it. We can't just rip it off like that. We need to be backwards compatible.